nest/02.introduction.md

```diff
@@ -0,0 +1,52 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all the files required to install, uninstall and update a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage packages**.
+It is similar to an app store for a smartphone, or the Windows Store for example.
+With Nest, it's up to the user to upgrade their installed packages whenever they want.
+
+Currently, to use Nest, one must use its Command Line Interface (CLI).
+This might sound intimidating for someone who has never seen a terminal or typed any command before, but if that's the case, don't worry! This guide is here to help you feel at home.
+
+## 1.2 Packages and dependencies
+It is common for a package to require other ones to work properly.
+For example, let's take a software that allows its users to send messages to each other, over the Internet.
+That software will need a way to send messages over the Internet, so either it has its own way to do it, either it will use an existing library for that purpose.
```

Comment:
```suggestion
That software will need a way to send messages over the Internet. To do so it will either implement it's own solution, or use an existing library for that purpose.
```